src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs

```diff
@@ -20,26 +23,28 @@ public class WhenIGetTrustedEmployers
         public const uint ExpectedUkPrn = 12345;
 
         private GetTrustedEmployersQueryHandler _handler;
-        private Mock<IProviderPermissionsService> _providerPermissionsService;
+        private Mock<IProviderService> _providerService;
         private GetTrustedEmployersQuery _query;
-        private IList<Employer> _expectedEmployers;
+        private IList<AccountLegalEntity> _expectedEmployers;
```

Comment:
worth renaming the var to `_expectedAccountLegalEntities`, as well?

Reply:
Updated